modules/_labs/tabs/react/lib/cursor/useCursorModel.tsx

```diff
@@ -114,7 +115,7 @@ export const useCursorModel = (config: CursorModelConfig = {}): CursorModel => {
   const [cursorId, setCursorId] = React.useState('');
   const initialCurrentRef = React.useRef('');
   const list = useListModel({
-    ...config,
+    ...(config as ListModelConfig),
```

Comment:
Narrowing the type here prevents `strictFunctionTypes` errors on the `...config` object having covariant errors between `CursorModelConfig` and `ListModelConfig` where `config` technically has callback and guard types that accept `CursorState` instead of `ListState`. Bivariance says this is okay since `CursorState` extends from `ListState`, but contravariance says it isn't. This casting prevents the need for bivariance hacks. A type error would occur if the casting was invalid, but I think this is a valid tradeoff.

Reply:
And if you don't fully understand covariance, contravariance, and bivariance, that's okay. I don't either.

Reply:
More info here: https://www.stephanboyer.com/post/132/what-are-covariance-and-contravariance